### ISSUE TITLE
treat this as a classification problem, not regression

### ISSUE THREAD

**Issue by mynameisvinn** [ansrivas/keras-rest-server#3]
since youre predicting 1/0 (ie this is a classification example) for xor, you should use `binary_crossentropy` for your loss function, not `mse`. 

https://github.com/ansrivas/keras-rest-server/blob/9aa819b3ffb6a76187b73185ec932adf67628e0b/createpickles.py#L68

additionally, you should use `softmax` as your activation function, not `linear`. this will return a predicted class (eg 0, 1) and not a real continuous value. 

https://github.com/ansrivas/keras-rest-server/blob/9aa819b3ffb6a76187b73185ec932adf67628e0b/createpickles.py#L62

**Comment by ansrivas** [comment]
Indeed you are right but this example was created with an aim to show that a trained model can be served over rest. 

For queries regarding XOR check this here : https://github.com/fchollet/keras/issues/112

**Comment by mynameisvinn** [comment]
yep, just friendly suggestion! thanks for the helpful example.
